src/test/java/com/epam/tc/hw2/TestExercise2HomePage.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class TestExercise2HomePage {
+
+    private WebDriver webDriver;
+    private final String expectedPageTitle = "Home Page";
+    private final String expectedUserName = "ROMAN IOVLEV";
+    private final Map<String, String> expectedLogNameStatus = Map.of(
+        "Water", "true",
+        "Wind", "true",
+        "metal", "Selen",
+        "Colors", "Yellow");
+    private final String expectedSelectColor = "Yellow";
+    private final int orderOfNameWordInLog = 1;
+    private final int orderOfStatusWordInLog = 5;
+
+
+    @BeforeClass
+    void setupWebDriver() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
+    void createNewDriver() {
```

Comment:
see the previous comment about abstract class

Reply:
Done